commercetools-models/src/main/java/io/sphere/sdk/products/search/LocaleSelection.java

```diff
@@ -0,0 +1,21 @@
+package io.sphere.sdk.products.search;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Parameters to select prices in {@link ProductProjectionSearch}.
+ *
+ * Use {@link LocaleSelectionBuilder} or {@link LocaleSelectionDsl} to create an instance.
+ *
+ *
+ */
+public interface LocaleSelection {
+    @Nullable
+    List<String> getLocaleProjection();
+
+    static LocaleSelectionDsl of(final String localeProjection)
```

Comment:
please add an additional method which allows the usage of List<String> localeProjections